packages/breadboard-web/src/boards/simplest.ts

```diff
@@ -4,42 +4,44 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Schema, base, board } from "@google-labs/breadboard";
-import { core } from "@google-labs/core-kit";
+import {
+  annotate,
+  board,
+  input,
+  object,
+  output,
+} from "@breadboard-ai/build";
+import { invoke } from "@google-labs/core-kit";
 
-const inputSchema = {
-  type: "object",
-  properties: {
-    text: {
-      type: "string",
-      title: "Prompt",
-      description: "The prompt to generate a completion for",
-      examples: ["Tell me a fun story about playing with breadboards"],
-    },
-  },
-  required: ["text"],
-} satisfies Schema;
+const text = input({
```

Comment:
Suggest calling this variable `prompt` (more specific than `text` and matches the `title`).